src/import-utils.ts

```diff
@@ -107,8 +107,16 @@ export async function inlineHtmlImport(
   rewriteAstBaseUrl(importAst, resolvedImportUrl, document.url);
 
   if (enableSourcemaps) {
+    const reparsedDoc = new ParsedHtmlDocument({
+      url: document.url,
+      contents: htmlImport.document.parsedDocument.contents,
+      ast: importAst,
+      isInline: document.isInline,
```

Comment:
Prefer `false` here over `document.isInline`, as this is definitely a top level doc, and without a location offset or an ast node it wouldn't be a valid inline doc anyways.